x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/constants.ts

```diff
@@ -7,9 +7,10 @@
 
 export const BUILT_IN_ID_PREFIX = 'builtin_';
 export const BUILT_IN_ALLOWED_INDICES = [
-  'apm-*',
+  'apm*',
```

Comment:
Check Soren's comment: https://github.com/elastic/kibana/pull/187021#discussion_r1667183693 

Do we really need apm index?

Reply:
good call, if similar data is available in either metrics/logs/traces then we can remove this